net/frr/src/opnsense/mvc/app/views/OPNsense/Quagga/ospf.volt

```diff
@@ -51,6 +51,14 @@ POSSIBILITY OF SUCH DAMAGE.
             }
         });
 
+        $("#{{formGridEditOSPFNeighbor['table_id']}}").UIBootgrid({
+            'search':'/api/quagga/ospfsettings/searchNeighbor',
```

Comment:
```suggestion
            'search':'/api/quagga/ospfsettings/search_neighbor',
```

the frr plugin does still need a style upgrade, but strictly speaking, camel should be converted to snake for endpoints. (same below)